### ISSUE TITLE
PipelineModel Docker Image Bind-to-port flag 

### ISSUE THREAD

**Issue by NMRobert** [aws/sagemaker-pytorch-container#140]
Hi,

I'm trying to create a PipelineModel which chains a SKLearn model into a PyTorch model, and I'm encountering the following error:

_ClientError: An error occurred (ValidationException) when calling the CreateModel operation: Your Ecr Image 763104351884.dkr.ecr.eu-west-1.amazonaws.com/pytorch-inference:1.2.0-cpu-py3 does not contain required com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true Docker label(s)._

My code roughly looks like the following:

```
preprocesor_estimator = SKLearn.attach('deepevent-feature-processor-model-05g').create_model()

de_estimator= sagemaker.pytorch.PyTorchModel(model_data='s3://<path-to-my-model-data>',
                              role=role,
                              entry_point='de_model.py',
                              image='763104351884.dkr.ecr.eu-west-1.amazonaws.com/pytorch-inference:1.2.0-cpu-py3',
                              framework_version='1.2.0',
                              sagemaker_session=sagemaker_session)

model_name = 'Model-inference-pipeline-' + timestamp_prefix
endpoint_name = 'Model-inference-pipeline-ep-' + timestamp_prefix

sm_model = PipelineModel(
    name=model_name, 
    role=role, 
    models=[
        preprocesor_estimator, 
        de_estimator])

sm_model.deploy(initial_instance_count=1, instance_type='ml.c4.xlarge', endpoint_name=endpoint_name)
```

Now, this is pretty confusing because as best as I can tell, the required bind-to-port flag _is_ set [see here](https://github.com/aws/sagemaker-pytorch-container/blob/d7d5801c742bf6469fe6779270c1c245c0ad81b1/docker/1.2.0/py3/Dockerfile.cpu#L5) Any guidance would be appreciated, thank you.

**Comment by NMRobert** [comment]
I've discovered that this does work with the pytorch inference 1.1.0 image - just not with the 1.2.0 image. I suspect that this might be an issue with the config of the container in ECR.

**Comment by akartsky** [comment]
Sorry for the late response 

The new 1.3.1 pytorch containers have bind-to-port=true LABEL
https://github.com/aws/sagemaker-pytorch-serving-container/tree/master/docker/1.3.1/py3
